components/cluster/libesmtp/Makefile

```diff
@@ -13,42 +13,41 @@
 # Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
 #
 
```

Comment:
This one doesn't publish here:
usr/openssl/1.1/lib/64 -lssl -lcrypto -lsocket  -pthread -pthread -pthread -m64   -pthread
ld: fatal: file .libs/client-login.o: wrong ELF class: ELFCLASS32
ld: fatal: file processing errors. No output written to .libs/sasl-login.so
collect2: error: ld returned 1 exit status

Reply:
I forgot that I have built with gcc-10... some flag missing.